app/models/repost.rb

```diff
@@ -0,0 +1,38 @@
+# rubocop:disable Metrics/LineLength
+# == Schema Information
+#
+# Table name: reposts
+#
+#  id         :integer          not null, primary key
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#  post_id    :integer          not null, indexed
+#  user_id    :integer          not null, indexed
+#
```

Comment:
do we want to have a unique constraint on these so you can only repost something once?